scripts/release.ts

```diff
@@ -33,53 +30,68 @@ const packages = {
 yargs(process.argv.slice(2))
   .scriptName("release")
   .usage("$0 <cmd> [args]")
+  .option("target-repo", {
+    describe: "Select upstream GitHub repository",
+    nargs: 1,
+    default: "web-platform-dx/feature-set",
+  })
   .command({
-    command: "init",
+    command: "init <semverlevel>",
     describe: "Start a new release pull request",
     builder: (yargs) => {
       return yargs
         .positional("semverlevel", {
           describe: "the Semantic Versioning level for the release",
-          choices: ["major", "minor", "patch", "prerelease"],
+          choices: semverChoices,
           default: "patch",
         })
         .demandOption("semverlevel", "You must provide a semver level");
     },
     handler: init,
   })
   .command({
-    command: "update",
+    command: "update <pr>",
     describe: "Update an existing release pull request",
     builder: (yargs) => {
       return yargs
         .positional("pr", {
-          describe: "the PR to rebase and update",
+          describe: "the PR (URL, number, or branch) to rebase and update",
+          type: "string",
         })
-        .demandOption("pr");
+        .option("bump", {
+          describe: "Update the Semantic Versioning level for the release",
+          nargs: 1,
+          choices: semverChoices,
+        })
+        .option("base", {
+          describe: "Branch to rebase against",
+          type: "string",
+          default: "main",
+        });
     },
     handler: update,
   })
   .command({
     command: "publish",
     describe: "Publish the package to npm",
     builder: (yargs) => {
-      return yargs
-        .positional("pr", {
-          describe: "the PR to rebase and update",
-        })
-        .demandOption("pr");
+      return yargs.option("dry-run", {
+        type: "boolean",
+        describe: "Do everything short of publishing",
+        default: true,
+      });
     },
     handler: publish,
   }).argv;
 
 function init(args) {
-  preflight();
+  preflight({ expectedBranch: "main" });
 
   const diff = diffJson();
 
   // Start a release branch
   // Convention borrowed from https://github.com/w3c/webref/blob/60ebf71b9d555c523975cfefb08f5420d12b7293/tools/prepare-release.js#L164-L165
-  const releaseBranch = `release-${new Date()
+  const releaseBranch = `release/web-features/${new Date()
```

Comment:
This groups the branches generated by this process. I thought this might help avoid name collisions (which I kept tab completing my way into 🙄).